File: Sheet #1 (Data type - Conditions)/C++/W-Mathematical Expression.cpp
#include <iostream>
using namespace std;

int main()
{
    long long int a, b, c;
    char q, s;

    cin >> a >> s >> b >> q >> c;

    if (s == '+')
    {
        if ((a + b) == c)
            cout << "Yes" << endl;
        else
            cout << a + b;
    }

    if (s == '-')
    {
        if ((a - b) == c)
            cout << "Yes" << endl;
        else
            cout << a - b;
    }

    if (s == '*')
    {
        if ((a * b) == c)
            cout << "Yes" << endl;
        else
            cout << a * b;
    }

    return (0);
}

File: Sheet #1 (Data type - Conditions)/python/W-Mathematical Expression.py
#!/usr/bin/python3

"""
Given a mathematical expression. The expression will be one of the following expressions:

A + B = C, A - B = C and A * B = C

where A, B, C are three numbers, S is the sign between A and B, and Q the '=' sign
"""
# A + B = C
user_input = input("")
user_input = user_input.split()
A = float(user_input[0])
op = user_input[1]
B = float(user_input[2])
res = float(user_input[4])

if op == '+':
    if A + B == res:
        print("Yes")
    else:
        print(int(A + B))

if op == '-':
    if A - B == res:
        print("Yes")
    else:
        print(int(A - B))

if op == '*':
    if A * B == res:
        print("Yes")
    else:
        print(int(A * B))

